src/consent-string.js

```diff
@@ -98,20 +99,33 @@ class ConsentString {
     if (updateDate === true) {
       this.lastUpdated = new Date();
     }
+    const fields = this.getStringFields(true);
+    return encodeConsentString(fields);
```

Comment:
Can we keep the two lists of field separated instead of having the `getStringFields` function? I don't think it brings a lot of value and makes reading the code a little harder I think.